DAB.md

```diff
@@ -705,8 +705,8 @@ type ListSupportedSystemSettingsRequest = DabRequest
 
 ```typescript
 interface OutputResolution {
-   width: int;
-   height: int;
+   width: number;
```

Comment:
This change makes sense as the current interface definitions are written in TypeScript. I wonder though if the specification should be expressed using some IDL language which provides more specific data types (i.e. floating-point number for resolution doesn't make much sense, so perhaps TS is too limited for expressing the intention).

Reply:
Will do